model/internal/cmd/pdatagen/internal/log_structs.go

```diff
@@ -88,6 +88,14 @@ var logRecord = &messageValueStruct{
 	description:    "// LogRecord are experimental implementation of OpenTelemetry Log Data Model.\n",
 	originFullName: "otlplogs.LogRecord",
 	fields: []baseField{
+		&primitiveTypedField{
+			fieldName:       "ObservedTimestamp",
+			originFieldName: "TimeUnixNano",
```

Comment:
I think this is wrong originFieldName

```suggestion
			originFieldName: "ObservedTimeUnixNano",
```

Reply:
Thanks, good catch.

Reply:
Still working on this. Regenerated did not pick up the change. Looking into it.

Reply:
Ok, fixed now.